docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -0,0 +1,194 @@
+
+If your Teleport cluster is configured using `second_factor:u2f`, you can opt to
+change it to `second_factor:webauthn` (allowing only WebAuthn authenticators) or
+`second_factor:on` (allowing both WebAuthn and OTP).
+
+It is recommended to capture the inferred `rp_id` in your `/etc/teleport.yaml`
+file. You can find the inferred `rp_id` in the Teleport Auth Server logs:
+
+```
+2021-10-04T18:03:22-07:00 INFO WebAuthn: RPID inferred from U2F configuration: "example.com" types/authentication.go:504
+```
+
+Example configuration, including both U2F and WebAuthn settings:
+
+```yaml
+# snippet from /etc/teleport.yaml:
+auth_service:
+  authentication:
+    type: local
+    # to enable WebAuthn support, set this field to 'on', 'optional' or 'webauthn'
+    second_factor: on
+    # valid, pre-existing U2F configuration
+    u2f:
+       app_id: https://example.com
+       facets:
+       - https://example.com:443
+       - https://example.com
+       - example.com:443
+       - example.com
+       device_attestation_cas:
+       - "/path/to/u2f_attestation_ca.pem"
+    # optional WebAuthn configuration
+    webauthn:
+      # inferred from the U2F app_id if not present.
+      rp_id: "example.com"
+      # copied from U2F device_attestation_cas if not present.
+      attestation_allowed_cas:
+      - "/path/to/u2f_attestation_ca.pem"
+      attestation_denied_cas:
+      - "/path/to/denied_ca.pem"
+```
+
+The fields in the above snippet are:
+
+- `rp_id` - public domain of the Teleport proxy, *excluding* protocol
+  (`https://`) and port number.
+  Inferred from U2F `app_id` if not present.
+
+- `attestation_allowed_cas` - optional allow list of certificate authorities (as
+  local file paths or in-line PEM certificate string) for [device verification](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Attestation.html).
+  This field allows you to restrict which device models and vendors you trust.
+  Devices outside of the list will be rejected during registration. By default
+  all devices are allowed.
+  If you must use attestation, consider using `attestation_denied_cas` to
+  forbid troublesome devices instead.
+  Copied from U2F `device_attestation_cas` if not present.
+
+- `attestation_denied_cas` - optional deny list of certificate authorities (as
+  local file paths or in-line PEM certificate string) for [device verification](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Attestation.html).
+  This field allows you to forbid specific device models and vendors, while
+  allowing all others (provided they clear `attestation_allowed_cas` as well).
+  Devices within this list will be rejected during registration. By default no
+  devices are forbidden.
+
+<h2 id="enable-webauthn">Enable WebAuthn support</h2>
+
+WebAuthn is disabled by default. To enable WebAuthn support, edit the Teleport
+configuration file `/etc/teleport.yaml` like so:
+
+```yaml
+# snippet from /etc/teleport.yaml:
+auth_service:
```

Comment:
This focuses on YAML. Can you provide alternative tctl resource edit for cloud in a tab item with a scope cloud

Reply:
Done here and in the migration section.

I went with "Static Config" and "Dynamic resources" for the tab labels and used ["oss","enterprise"] and ["cloud"] scopes, respectively.